gravitee-apim-console-webui/package.json

```diff
@@ -17,8 +17,8 @@
     "@fontsource/libre-franklin": "4.4.5",
     "@gravitee/ui-analytics": "4.3.0",
```

Comment:
WDYT to keep this dep with same version like `@gravitee/ui-particles-angular` & `@gravitee/ui-policy-studio-angular` 🙏  ?